docs/howto/screenshot.md

```diff
@@ -0,0 +1,29 @@
+# Create Screenshot and paste into an issue
+
+## Description
+The instructions below describe how to capture a screenshot of your screen, either your entire screen or a partial screenshot. These can be pasted into GitHub issues, pull requests or any markdown file. 
+
+## Screenshot Instructions (Mac)
+1. **Full screen**: Hit the Command, Shift and 3 keys together to take a screenshot of the entire screen
+
+![image](https://user-images.githubusercontent.com/6722114/181063086-d14b3e28-9d25-4464-b575-ae58b69ef395.png)
+
+1. **Partial screen**: Hit the Shift, Control, Command and 4 keys together to take a screenshot of a portion of the screen. Crosshairs will appear and select the portion you would like to capture.
+
+![image](https://user-images.githubusercontent.com/6722114/181063353-c96be768-0763-402b-8a0b-811ad41fb808.png)
+
+1. Paste the partial screenshot into comment box in GitHub.
+
+### Video explanation: Partial screenshot on a Mac
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/W1DWZbYQc7Y" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
this video does not appear properly in the preview but maybe because I just uploaded it to YouTube?